scss/prototype/_spacing.scss

```diff
@@ -0,0 +1,75 @@
+// Map containing all the `spacing` classes
+// @type Map
+$prototype-spacing-classes: (
+  margin: m, 
+  padding: p
+)!default;
```

Comment:
To whom so ever it may concern,

Spacing utilities share a different class convention then any other classes within this PR or the foundation framework
Eg, => `.m-t-10` 
Reason: It gets to head pretty quickly and easier for prototyping. Shall i keep it like or change it. 

Thoughts @natewiebe13